connector/connect/server/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -126,6 +126,20 @@ package object dsl {
           Expression.UnresolvedFunction.newBuilder().setFunctionName("min").addArguments(e))
         .build()
 
+    def proto_max(e: Expression): Expression =
```

Comment:
There is need to add `proto_` prefix? Just call it max?

Reply:
Same for below

Reply:
I just follows up the existing `proto_min`.